components/service/glean/src/main/java/mozilla/components/service/glean/ping/PingMaker.kt

```diff
@@ -44,6 +45,8 @@ internal class PingMaker(
         pingInfo.put("ping_type", pingName)
         pingInfo.put("telemetry_sdk_build", BuildConfig.LIBRARY_VERSION)
 
+        pingInfo.put("experiments", ExperimentsStorageEngine.getSnapshotAsJSON("", false))
```

Comment:
Let's comment on why we need to special case the `ExperimentsStorageEngine` here.